inventory/byo/hosts.origin.example

```diff
@@ -571,10 +571,18 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # network blocks should be private and should not conflict with network blocks
 # in your infrastructure that pods may require access to. Can not be changed
 # after deployment.
+#
+# WARNING : Do not pick subnets that overlap with the default Docker bridge subnet of
+# 172.16.0.0/16.  Your installation will fail and/or your configuration change will
+# cause the Pod SDN or Cluster SDN to fail.
+#
+# WORKAROUND : If you must use an overlapping subnet, you can relocate the Docker
+# default subnet by doing the following before installing OCP:
+#     $ echo 'DOCKER_NETWORK_OPTIONS="--bip=192.168.2.1/24"' >> /etc/sysconfig/docker-network
```

Comment:
The concern is that this would overwrite anything else in DOCKER_NETWORK_OPTIONS. Ideally everyone would mentally interpret that and go edit the file by hand but if we give someone something to copy and paste then there's a good change they'll just do that and won't notice they've just blown away any other settings in that variable already.

How about lets just say "Configure a non conflicting docker0 CIDR range by adding '--bip=192.168.2.1/24' to DOCKER_NETWORK_OPTIONS environment variable located in /etc/sysconfig/docker-network" ?